cmd/config.go

```diff
@@ -203,6 +202,18 @@ func initConfig(cmd *cobra.Command) error {
 		config.DefaultChain = defaultChain
 	}
 
+	if cmd.PersistentFlags().Changed("output") {
+		output, err := cmd.PersistentFlags().GetString("output")
+		if err != nil {
+			return err
+		}
+
+		// Should output be a global configuration item?
```

Comment:
The purpose of the `client` library is to make it easy for devs to build cosmos bots so I'm fine doing it this way